libretro/RetroArch
# Display a "Fast forward."  message

Similar to how we have a "Slow motion." message appear, display a "Fast forward." message when sped up.

![fastforward](https://user-images.githubusercontent.com/25086/32816492-f256bfae-c986-11e7-9cb1-53255ef85c7d.png)

Fixes #5667 

## QA

1. Compile
2. Run a game
3. Press Fast Forward key (default is l)
4. See "Fast forward." appear on the screen
